.gitignore

```diff
@@ -10,6 +10,9 @@
 # Compiled python code.
 *.pyc
 
+# Distribution / packaging
+*.egg-info/
```

Comment:
Also, you may find [these make targets](https://github.com/SatelliteQE/nailgun/blob/master/Makefile#L28-L32) useful. `bdist_wheel --universal` is only applicable if you're generating and distributing a Python 3 package.